packages/manager/modules/cloud-universe-components/src/price/service.js

```diff
@@ -7,32 +7,49 @@ export default class OvhCloudPriceHelper {
   /* @ngInject */
   constructor(
     $q,
+    $http,
     CucCurrencyService,
     cucUcentsToCurrencyFilter,
     OvhApiCloudProject,
     OvhApiMe,
     OvhApiOrderCatalogPublic,
   ) {
     this.$q = $q;
+    this.$http = $http;
     this.CucCurrencyService = CucCurrencyService;
     this.cucUcentsToCurrencyFilter = cucUcentsToCurrencyFilter;
     this.OvhApiOrderCatalogPublic = OvhApiOrderCatalogPublic;
     this.OvhApiCloudProject = OvhApiCloudProject;
     this.OvhApiMe = OvhApiMe;
   }
 
-  getPrices(serviceName) {
+  getMe() {
```

Comment:
coreConfig can be used here

```suggestion
coreConfig.getUser()
```